src/DataFramesMeta.jl

```diff
@@ -387,17 +386,29 @@ function transform(d::Union{AbstractDataFrame, AbstractDict}; kwargs...)
     return result
 end
 
+
 function transform(g::GroupedDataFrame; kwargs...)
     result = DataFrame(g)
     idx2 = cumsum(Int[size(g[i],1) for i in 1:length(g)])
     idx1 = [1; 1 + idx2[1:end-1]]
```

Comment:
I think `idx1` and `idx2` are not needed?

Reply:
true, sorry.